OpenUpSA/wazimap-ng
# added chart type configuration

## Description
added a dropdown to select the chart type

![image](https://user-images.githubusercontent.com/53019884/178903001-f38a0589-bbdc-4246-93dc-0d7c9e312228.png)


## Related Issue
https://wazimap.atlassian.net/browse/WNCM-409

## How to test it locally
* go to any profile indicator page 
* change the chart type using the new dropdown
* on the ui confirm that the indicator is displayed in the selected chart type

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
